### DISCUSSION TITLE
Removing the dir .git, the git branch icon remains in the terminal, how to proceed?

### DISCUSSION THREAD

**Discussion by zThiagoR** [JanDeDobbeleer/oh-my-posh#2849] [Troubleshoot] [answered]
Goodnight,

I went to install oh-my-posh for the first time using it, and I was trying out its features, and when I started testing with git, when you give the command `rmdir .git` in the project, the branch icon remains there.

How can it be resolved?

![image](https://user-images.githubusercontent.com/56285370/192894724-89cf1e20-827c-49af-b90c-dc834a636176.png)

**Comment by JanDeDobbeleer** [comment, marked as answer]
@zThiagoR are you using and have enabled posh-git here?

**Reply by zThiagoR** [reply]
Yes, sir.

**Reply by JanDeDobbeleer** [reply]
@zThiagoR can you show me the output of `Get-GitStatus` in this scenario?

**Reply by zThiagoR** [reply]
![image](https://user-images.githubusercontent.com/56285370/193134160-e8155cea-683d-4156-a5e7-447fa5de9807.png)

**Reply by JanDeDobbeleer** [reply]
@zThiagoR in this case you'll need to ask over at posh-git as I wouldn't expect them to report this as a git directory. You deleted that .git folder, right?

**Reply by zThiagoR** [reply]
@JanDeDobbeleer I managed to dig deeper into the error, the error it caused was that it couldn't delete the empty folder and I had to use param -force to force removal of the folder which solved my case.

However, thank you for trying to help me. 😅😅
